app/src/main/java/live/hms/app2/ui/meeting/MeetingFragment.kt

```diff
@@ -800,6 +802,73 @@ class MeetingFragment : Fragment() {
     }
   }
 
+  private val pipReceiver by lazy { object : BroadcastReceiver() {
```

Comment:
Would be good if the pip receiver was a separate class and took functions to leave meetings, toggle local audio etc